.github/workflows/docstring-md.yml

```diff
@@ -0,0 +1,37 @@
+name: Build Docs and Open PR
+on:
+  push:
+    branches:
+        - 'future3/**' #set to future3/develop after testing
+    paths:
+        - '**.py'
+  # remove after testing
+  pull_request:
+    branches:
+        - 'future3/**'
+    paths:
+        - '**.py'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Generate Python Docs
+      uses: s-martin/markdown-docs@master
+      with:
+        src_path: src
+        dst_path: documentation/developers/docstring
+    - name: Fix permissions
+      run: |
+        sudo chown -R $USER:$USER .
```

Comment:
Why is that needed?

Reply:
See here in an old run.
after the action did run for at least one time you were not able to unlink the previous files: https://github.com/s-martin/RPi-Jukebox-RFID/actions/runs/7147018267/job/19465782164#step:5:1

The chown is a workaround